apps/fishing-map/features/user/UserInfo.tsx

```diff
@@ -2,12 +2,13 @@ import { useCallback, useState } from 'react'
 import { useSelector } from 'react-redux'
 import { Trans, useTranslation } from 'react-i18next'
 import { Button, Spinner } from '@globalfishingwatch/ui-components'
+import { GUEST_USER_TYPE } from '@globalfishingwatch/api-client'
```

Comment:
I like this! does it makes sense to move [this other ones](https://github.com/GlobalFishingWatch/frontend/pull/1546/files#diff-dd7c89a932640c2a94ce168c7f83787f30219bd4e6493c5e318a9f3966d92f7bR25) too?

Reply:
I moved this because I needed to update the API url, maybe we can the others in another PR